gomanifest/build.go

```diff
@@ -28,9 +29,15 @@ func main() {
 		os.Exit(2)
 	}
 
+	// Extract go executable path, if given
+	goExec := "go"
+	if argsLen == 4 {
+		goExec = os.Args[3]
```

Comment:
Can you check the existence of goExec path and throw error on invalid path? (os.Stat)